prac2-pages/chhotu/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MaterialApp(
+      home: Project(),
+    ));
+
+class Project extends StatefulWidget {
+  const Project({super.key});
+
+  @override
+  State<Project> createState() => _ProjectState();
+}
+
+class _ProjectState extends State<Project> {
+  bool value = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
```

Comment:
Try to wrap your entire content in **SafeArea** widget

Refer to the documentation to know what it is and how to use it :thought_balloon: